mediapipe/tasks/ios/core/sources/MPPExternalFile.m

```diff
@@ -0,0 +1,27 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ ==============================================================================*/
+#import "mediapipe/tasks/ios/core/sources/MPPExternalFile.h"
+
+@implementation MPPExternalFile
+
+- (id)copyWithZone:(NSZone *)zone {
+  MPPExternalFile *externalFile = [[MPPExternalFile alloc] init];
+
+  externalFile.filePath = self.filePath;
```

Comment:
Do we need to do a copy here?

Reply:
See my response [here](https://github.com/google/mediapipe/pull/3911#discussion_r1044094752)